operations/helm/tests/test-enterprise-federation-frontend-values-generated/mimir-distributed/templates/minio/create-bucket-job.yaml

```diff
@@ -4,44 +4,3 @@
 # however the hook won't be executed if helm install is run
```

Comment:
I wonder if the reason this file is being generated is because the `if` is after the comment in https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/templates/minio/create-bucket-job.yaml.

Have you tried moving the `{{- if .Values.minio.enabled }}` to the top of the file?

(not a big deal if we can't fix this easily)

Reply:
oh, wow i didn't think to actually look in the file

Reply:
should be fixed in https://github.com/grafana/mimir/pull/9673/commits/f9cd475715127c043ca5c22251208e22381abb79